release/release_data_tests.yaml

```diff
@@ -1,5 +1,5 @@
 - name: DEFAULTS
-  group: data-tests
+  group: :database: Data tests
```

Comment:
Looks good, though I don't have the context of the change.

Reply:
There's not any context. I just thought this might look nicer